plutus-use-cases/hie.project

```diff
@@ -0,0 +1,4 @@
+packages: .
+
+package plutus-use-cases
+  ghc-options: -fplugin-opt Language.PlutusTx.Plugin:skip
```

Comment:
I am going to double check this, but from memory the error I was getting was related to unfoldings.

Reply:
That suggests to me that `ghcide` is doing the wrong thing (which it may well be!). If we can't fix it upstream this is fine, I guess.

Generally I'm not sure how well having IDE-specific `ghc-options` is going to go. Consider a dev workflow:
- Work on project for a bit with IDE
- Run `cabal test` to check tests

But these now use different `ghc-options`, so they'll reconfigure (and therefore rebuild) the whole project!

Reply:
I set it up so it also use `--builddir dist-hie` so that the hie object files do not mix with non hie ones.

https://github.com/mpickering/hie-bios/compare/master...hamishmack:hkm/hie-project-file#diff-d9d44b1f07a4384f5b7934a7591ea3faR429